### PR TITLE
Feature/backport requirements to stufe2 ptdata 974

### PR DESCRIPTION
**Pull request by f-peverali**
# Contributor Pull Request

## Description
soften requirements for external ressource import

## Motivation and Context
solves PTDATA-974

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this IG / specification.
- [ ] My change requires a change to the documentation or narrative (intend) of the IG.
- [ ] I have already updated the documentation / narrative (intend) accordingly.

## Reviewer / Quality Assurance Checklist

- [ ] The Present PR does not need a thorough review process, due to brevity, low complexity or because it represents a rather minor change; otherwise go trough the list! ... and read the linked manual, if you did not do it yet!

### content review steps ([based on Best Practice IG Germany](https://simplifier.net/guide/Best-Practice-bei-der-Implementierung-und-Spezifizierung-mit-HL7/%C3%9Cbersicht/Spezifikation?version=current))
- [x] no content review needed
- [ ] [cardinalities in profiles](https://simplifier.net/guide/Best-Practice-bei-der-Implementierung-und-Spezifizierung-mit-HL7/%C3%9Cbersicht/Spezifikation/Profilierung/Umgang-mit-Kardinalit%C3%A4ten.page.md?version=current) are added with caution (specially 'max') and are thoroughly motivated in a comment
- [ ] modifying-Elements are [highlighted](https://simplifier.net/guide/Best-Practice-bei-der-Implementierung-und-Spezifizierung-mit-HL7/%C3%9Cbersicht/Spezifikation/Profilierung/Umgang-mit-Modifying-Elements.page.md?version=current)
- [ ]  [meta.profile Elements](https://simplifier.net/guide/Best-Practice-bei-der-Implementierung-und-Spezifizierung-mit-HL7/%C3%9Cbersicht/Spezifikation/Profilierung/Umgang-mit-Meta-Daten.page.md?version=current) not added or added with extreme caution
- [ ] [StructureDefinition.description](https://simplifier.net/guide/Best-Practice-bei-der-Implementierung-und-Spezifizierung-mit-HL7/%C3%9Cbersicht/Spezifikation/Profilierung/Dokumentation-und-Nachvollziehbarkeit.page.md?version=current) is filled
- [ ] [Constrained Elements](https://simplifier.net/guide/Best-Practice-bei-der-Implementierung-und-Spezifizierung-mit-HL7/%C3%9Cbersicht/Spezifikation/Profilierung/Dokumentation-und-Nachvollziehbarkeit.page.md?version=current) contain
    - [ ] naming in ElementDefinition.short 
    - [ ] ElementDefinition.comment contains a motivation
- [ ] [Capability Statement](https://simplifier.net/guide/Best-Practice-bei-der-Implementierung-und-Spezifizierung-mit-HL7/%C3%9Cbersicht/Spezifikation/Erstellung-eines-CapabilityStatements.page.md?version=current) exists 
    - [ ] the textual searchparameters of every instance do correspond to the definition within the CapabilityStatement
- [ ] [relevant examples exist](https://simplifier.net/guide/Best-Practice-bei-der-Implementierung-und-Spezifizierung-mit-HL7/%C3%9Cbersicht/Spezifikation/Erstellung-von-Beispieldaten.page.md?version=current)

### formal intellectual review steps of [Implementation Guide](https://simplifier.net/guide/Best-Practice-bei-der-Implementierung-und-Spezifizierung-mit-HL7/%C3%9Cbersicht/Spezifikation/Erstellung-eines-Implementierungsleitfadens.page.md?version=current) 
- [x] no formal review needed
- [ ] proofreading for orthography 

- [ ] there are no (critical) validation Errors in the CI pipeline 
    - [ ]  inspected and no critical errors found(In the "github action view" -> Option button -> "View raw logs") - possibly list non-critical below
- [ ] skim reading for correct rendering of IG (possibly using IG-grep of string String like "Could not render")

### automated review steps:
- [ ] no formal review needed
- [ ] check IG-pages for broken links (possibly using plug-in)
- [ ] 

### For reviewers
You migh use this: https://conventionalcomments.org/

### CODE REVIEW

File: ImplementationGuide/markdown/UebergreifendeFestlegungen/UebergreifendeFestlegungen_Rest.md [hunk 1]
```diff
@@ -30,18 +32,20 @@ json
...
+Das Update einer Ressource KANN per HTTP PUT (vgl. [FHIR RESTful API - update](https://www.hl7.org/fhir/R4/http.html#update)) unterstützt werden. Es ist zu beachten, dass beim Update einer Ressource bestimmte dazugehörige [Metadaten](https://www.hl7.org/fhir/R4/resource.html#Meta) beibehalten werden SOLLTEN. Die gleichen Vorgaben für die Handhabung von invaliden Ressourcen wie beschrieben im Abschnitt "Create-Interaktionen", gelten auch für Update-Interaktionen.
 
 
 ## Sicherheitsaspekte
 Alle REST-Interaktionen müssen sowohl mittels HTTP als auch HTTPS (TLS-Verschlüsselung) unterstützt werden. Vorgaben zur TLS-Verschlüsselung sind dem nachfolgenden Link für die FHIR Security Check List zu entnehmen.
 Im Echtbetrieb MUSS die Kommunikation ausschließlich per HTTPS erfolgen.
 Weiterhin sind geeignete Maßnahmen zur Risiko-Minimierung (z.B. Benutzerautorisierung / -authentifikation) zu treffen, siehe http://build.fhir.org/security.html#6.1.0. 
-Diese sind in Stufe 1 des ISiK Basismoduls jedoch nicht bestätigungsrelevant.
\ No newline at end of file
+Diese sind in Stufe 1 des ISiK Basismoduls jedoch nicht bestätigungsrelevant.
```
Comment (alexzautke) [inline ImplementationGuide/markdown/UebergreifendeFestlegungen/UebergreifendeFestlegungen_Rest.md]: Stufe 1? Ich denke das sollte angepasst werden.

**Review by alexzautke — CHANGES_REQUESTED** [review]

File: ImplementationGuide/markdown/UebergreifendeFestlegungen/UebergreifendeFestlegungen_Rest.md (hunk 1 above)
Comment (f-peverali) [reply ImplementationGuide/markdown/UebergreifendeFestlegungen/UebergreifendeFestlegungen_Rest.md]: suggestion: würde den Satz an der Stelle entfernen.
todo: in Stufe 3 genauso

Comment (f-peverali) [reply ImplementationGuide/markdown/UebergreifendeFestlegungen/UebergreifendeFestlegungen_Rest.md]: hier für Stufe 3 PR: https://github.com/gematik/spec-ISiK-Basismodul/pull/472

Comment (f-peverali) [reply ImplementationGuide/markdown/UebergreifendeFestlegungen/UebergreifendeFestlegungen_Rest.md]: in Stufe 4 war es bereits angepasst

Comment (f-peverali) [inline ImplementationGuide/markdown/UebergreifendeFestlegungen/UebergreifendeFestlegungen_Rest.md]:
```suggestion
Diese sind in Stufe 2 des ISiK Basismoduls jedoch nicht bestätigungsrelevant.
```

Comment (f-peverali) [reply ImplementationGuide/markdown/UebergreifendeFestlegungen/UebergreifendeFestlegungen_Rest.md]: habe einfach auf "Stufe 2" gesetzt

**Pull request merged**